src/components/Input.module.css

```diff
@@ -0,0 +1,41 @@
+.form {
+  display: flex;
+  width: 90%;
+  max-width: 40rem;
+  margin: 2rem auto;
+  margin-bottom: 0.5rem;
+}
+
+.form input {
+  display: block;
+  width: 100%;
+  padding: 0.5rem;
+  font: inherit;
+  font-size: 1.5rem;
+  border-radius: 4px;
+  border: none;
+  border-bottom: 2px solid #2a6b6a;
+  background-color: #f7f5ef;
+}
+
+.form input:focus {
+  outline: 2px solid #2a6b6a;
+}
+
+.form button {

```

Comment:
I prefer to create classes instead of nesting
.form button
.form input
.form input:focus